### ISSUE TITLE
Error 502

### ISSUE THREAD

**Issue by skilver-io** [littlebizzy/slickstack#70]
Hi all,

I am running into HTTP 502 Bad Gateway Errors on a few of my latest slickstack instances. At the same time I am also starting to run into 
(1) login issue via the `/wp-admin/` page (redirecting to `wp-login.php?redirect_to=https....`
(2) caching issues with my cart pages of my WooCommerce plugin

I have noticed earlier today that your littlebizzy website was also not accessable as well due to an 500 error. I believe these issues came up with the slickstack updates I did yesterday and the changes in terms of the staging and dev environment. I am running multiple ecommerce pages on slickstack and customers aren't able to go through the checkout process due to the caching issues =/

Is this something on your side in regard to nginx? If so, are you guys aware of the issues and working on a fix atm?

Would be highly appreciating an answer.

Best
Dennis

**Comment by jessuppi** [comment]
Thanks for reporting @skilver-io 

Yes, we had some issues today with cookies and logins, lesson (partially) learned! We had attempted to optimize `COOKIE_DOMAIN` inside `wp-config.php` not realizing it would cause browser cache conflicts for some websites, so we went ahead and commented out our previous changes to the template.

At the same time, WordPress released two security patches, 5.5.2 and 5.5.3, adding to confusion.

We rolled back the `wp-config.php` template, force-updated it using `ss-worker` and force updated WordPress to the latest version and then reset permissions for all SS users. This combination of events caused confusion for some users and on very large sites the permissions took a long time to reset, causing brief outages as well.

Using our `xxx-common.php` MU plugin we've added this message to the WP Admin login page for now:

> Having trouble logging in? Please clear your browser cache and/or cookies for this site and try again. Thanks, and apologies! — SlickStack

TLDR it all comes down to a cached cookie conflict in your browser, but only for some users (probably depending on their browsing history or level of recent activity on your SS website).

Please let us know if any continued issues, I believe this should be resolved for most servers.
